src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -0,0 +1,266 @@
+        const canTransfer = transferAmount > Fee;
+        const isButtonDisabled = !canTransfer || !selectedAccount;
+
+        if (selectedAccount) {
+            const iconProperties = PaymentUtils.getPaymentMethodIconProperties(selectedAccount);
+            selectedAccount.icon = iconProperties.icon;
+            selectedAccount.iconSize = iconProperties.iconSize;
+        }
+
+        return (
+            <ScreenWrapper>
+                <KeyboardAvoidingView>
+                    <HeaderWithCloseButton
+                        title={this.props.translate('common.transferBalance')}
+                        shouldShowBackButton
+                        onBackButtonPress={() => Navigation.goBack()}
+                        onCloseButtonPress={() => Navigation.dismissModal(true)}
+                    />
+                    <View style={[styles.flex1, styles.flexBasisAuto, styles.justifyContentCenter]}>
+                        <CurrentWalletBalance balanceStyles={[styles.text7XLarge]} />
+                    </View>
+                    <ScrollView style={styles.flexGrow0} contentContainerStyle={styles.p5}>
+                        {_.map(this.paymentTypes, paymentType => (
+                            <MenuItem
+                                key={paymentType.key}
+                                title={paymentType.title}
+                                description={paymentType.description}
+                                iconWidth={variables.iconSizeXLarge}
+                                iconHeight={variables.iconSizeXLarge}
+                                icon={paymentType.icon}
+                                success={selectedPaymentType === paymentType.key}
+                                wrapperStyle={{
+                                    ...styles.mt3,
+                                    ...styles.pv4,
+                                    ...styles.transferBalancePayment,
+                                    ...(selectedPaymentType === paymentType.key
+                                        && styles.transferBalanceSelectedPayment),
+                                }}
+                                // eslint-disable-next-line react/no-unused-state
+                                onPress={() => {
+                                    PaymentMethods.updateWalletTransferData({filterPaymentMethodType: paymentType.type});
+                                    Navigation.navigate(ROUTES.SETTINGS_PAYMENTS_CHOOSE_TRANSFER_ACCOUNT);
+                                }}
+                            />
+                        ))}
+                        <ExpensifyText
+                            style={[styles.pv5, styles.textStrong, styles.textLabel, styles.justifyContentStart]}
+                        >
+                            {this.props.translate('transferAmountPage.whichAccount')}
+                        </ExpensifyText>
+                        {!!selectedAccount
```

Comment:
double bang are not widely used in our codebases. Please do not use this syntax.